kubernetes/manifest-templates/velum.yaml

```diff
@@ -82,3 +90,6 @@ spec:
     - name: velum-source-code
       hostPath:
         path: ${project_dir}
+    - name: velum-secrets
+      hostPath:
+        path: /tmp/velum-secrets
```

Comment:
It's OK to mount secrets on `/tmp` on the containers, but on the host we want this path to survive reboots, so our secrets are saved across restarts (otherwise we can lose sessions right now, or other secrets in the future)